### ISSUE TITLE
Age in MIMIC-IV

### ISSUE THREAD

**Issue by simonschallmoser** [eth-mds/ricu#43]
Hi everyone,

I was just wondering whether the age in MIMIC-IV should be adapted for patients with multiple ICU stays. As far as I understood the age is taken from the "anchor_age" column in the "patients" table and not modified for later ICU stays where the year in the "intime" column is not the same anymore as the "anchor_year" from the "patients" table. This can result in age differences of up to 12 years.

Am I correct and this should be adapted or did I miss something here?

Best,
Simon

**Comment by prockenschaub** [comment]
I agree with you. Would you feel confident enough with `ricu`'s inner workings to try and fix this in a PR? If you need it, I can give some guidance.

**Comment by simonschallmoser** [comment]
Sorry, I have never really worked with R, so I would leave this for someone with a bit more expertise than I have.
